cmd/buffer-api/main.go

```diff
@@ -147,10 +147,10 @@ func run() error {
 			docsFs := http.FS(openapi.Fs)
 			swaggerUiFs := http.FS(swaggerui.SwaggerFS)
 			endpoints := bufferGen.NewEndpoints(svc)
+			endpoints.Use(middleware.OpenTelemetryExtractEndpoint())
 			server := bufferGenSvr.New(endpoints, c.Muxer, c.Decoder, c.Encoder, c.ErrorHandler, c.ErrorFormatter, docsFs, docsFs, docsFs, docsFs, swaggerUiFs)
 
 			// Mount endpoints
-			server.Use(middleware.OpenTelemetryExtractEndpoint())
```

Comment:
Toto som mal blbo, su na to teste ale tie boli tiez blbo :) 

Tato cast ma trackovat Goa endpointy, teda nejaky vygenerovany kod s danym interface.
A v teste nechcem generovat nejake jednoduche Goa API, tak to musim nasimulovat rovnako, ako sa to deje v Goa.

Chcel som to mat vsetko ako HTTP middleware, tak som to registroval do server, ale zistil som, ze data v DD nemam, a je to potrebne registrovat ako Goa endpoint middleware, az tam su v `ctx` potrebne informacie.